Realm/RLMSyncUser.mm

```diff
@@ -253,6 +254,7 @@ + (void)_performLogInForUser:(RLMSyncUser *)user
                     return;
                 }
                 user->_user = sync_user;
+                user.isAdmin = model.refreshToken.tokenData.isAdmin;
```

Comment:
Is this state that should be tracked on object store's `SyncUser` class? I see that it already has  `SyncUser::is_admin()`, which represents something different than this flag.